log/log.go

```diff
@@ -108,6 +114,8 @@ func (l logLevel) String() string {
 		return "ERROR "
 	case levelDebug:
 		return "DEBUG "
+	case levelTrace:
+		return ""
```

Comment:
Could you point out why the level name is empty, or at least add a comment?

Reply:
And it is not configurable. Uses string template for debug logs: https://github.com/aws/aws-sdk-go/blob/dc8260e712a988dd3fb92e8f5f78240592c781ed/aws/client/logger.go#L14-L22

Reply:
it's much clear now, thank you.